app/moves/middleware/set-results.single-requests.js

```diff
@@ -7,6 +7,7 @@ async function setResultsSingleRequests(req, res, next) {
     req.results = await singleRequestService.getAll(req.body.requested)
     req.resultsAsTable = presenters.singleRequestsToTableComponent({
       query: req.query,
+      currentLocationType: req.session.currentLocation.location_type,
```

Comment:
We might not need to pass it in to this presenter, unless we're looking to add identifier to the table view below? @ElizabethGoddard?

<kbd>![image](https://user-images.githubusercontent.com/3327997/118248806-a28d5000-b49c-11eb-9e93-67cc0cb106ad.png)</kbd>

Reply:
Hi @Thource and @teneightfive thanks for bringing this up, it hadn't crossed my mind as I'm less familiar with single requests. I think that actually it makes sense to include it here too ultimately, since it's a key identifier. That said this isn't as much of a priority. Can we accommodate the code so that we can do it later?